### PR TITLE
Feedback form [#133400449]

### PR DESCRIPTION
**Pull request by IlkhamGaysin**

### CODE REVIEW

File: app/models/feedback.rb [hunk 1]
```diff
@@ -0,0 +1,10 @@
+class Feedback
+  VALID_EMAIL_REGEX = /\A[\w+\-.]+@[a-z\d\-.]+\.[a-z]+\z/i
```
Comment (dmitry-fedorov) [inline app/models/feedback.rb]: That regex allows invalid email addresses for example: `....@....a`

File: app/views/pages/index.html.slim [hunk 2]
```diff
@@ -3,4 +3,4 @@
 .container
   .row
     .flex-video.widescreen.columns
-      iframe width="560" height="315" src="https://www.youtube.com/embed/bZy8Nmf6f_Q" style="border:0" allowfullscreen
+      iframe width='560' height='315' src='https://www.youtube.com/embed/bZy8Nmf6f_Q' style='border:0' allowfullscreen
```
Comment (dmitry-fedorov) [inline app/views/pages/index.html.slim:6]: Prefer double quotes

**Review by dmitry-fedorov — CHANGES_REQUESTED** [review]

File: .env.example [hunk 3]
```diff
@@ -1,4 +1,11 @@
...
-# Port
+
+# Port to serve application
 PORT=5000
+
+# We send devise email using this "from" address
+MAILER_SENDER_ADDRESS=noreply@example.com
+
+# Feedback email
+FEEDBACK_RECEIVER_EMAIL=
```
Comment (AmirMingaliev) [inline .env.example]: Always add pattern of env variable

File: spec/features/visitor/create_feedback_spec.rb [hunk 4]
```diff
@@ -0,0 +1,23 @@
+require 'rails_helper'
+
+feature 'Create Feedback' do
+  let!(:feedback_attributes) { attributes_for(:feedback) }
```
Comment (AmirMingaliev) [inline spec/features/visitor/create_feedback_spec.rb]: I think we don't need bang sign here. A lazy `let` should be enough :)

**Review by AmirMingaliev — CHANGES_REQUESTED** [review]

File: .rubocop.yml [hunk 5]
```diff
@@ -59,3 +60,9 @@ Style/StringMethods:
   Enabled: true
 Rails/HasAndBelongsToMany:
   Enabled: false
+
+RSpec/MultipleExpectations:
+  Max: 7
+
+RSpec/ExampleLength:
+   Max: 12
```
Comment (AmirMingaliev) [inline .rubocop.yml]: indentation?

**Review by AmirMingaliev — APPROVED** [review]
lgtm 👍

**Review by dmitry-fedorov — APPROVED** [review]
Not sure about regex but ok :)

**Pull request merged**